platforms/chibios/drivers/uart.c

```diff
@@ -30,9 +32,16 @@ void uart_init(uint32_t baud) {
     if (!is_initialised) {
         is_initialised = true;
 
+#if defined(MCU_KINETIS)
+        serialConfig.sc_speed = baud;
+#else
         serialConfig.speed = baud;
+#endif
 
-#if defined(USE_GPIOV1)
+#if defined(MCU_KINETIS)
+        palSetLineMode(SD1_TX_PIN, SD1_TX_PAL_MODE);
+        palSetLineMode(SD1_RX_PIN, SD1_RX_PAL_MODE);
+#elif defined(USE_GPIOV1)
         palSetLineMode(SD1_TX_PIN, PAL_MODE_ALTERNATE_OPENDRAIN);
         palSetLineMode(SD1_RX_PIN, PAL_MODE_ALTERNATE_OPENDRAIN);
```

Comment:
Would be nice to handle this more like i2c, where we,

- keep the `USE_GPIOV1` path
- update `USE_GPIOV1` to use `SD1_TX_PAL_MODE`/`SD1_RX_PAL_MODE`
- Add a section that defaults that variable in `uart.h`
```c
#ifdef USE_GPIOV1
#    ifndef SD1_TX_PAL_MODE
#        define SD1_TX_PAL_MODE PAL_MODE_ALTERNATE_OPENDRAIN
#    endif
#    ifndef SD1_RX_PAL_MODE
#        define SD1_RX_PAL_MODE PAL_MODE_ALTERNATE_OPENDRAIN
#    endif
#else
// The default PAL alternate modes are used to signal that the pins are used for UART
#    ifndef SD1_TX_PAL_MODE
#        define SD1_TX_PAL_MODE 7
#    endif
#    ifndef SD1_RX_PAL_MODE
#        define SD1_RX_PAL_MODE 7
#    endif
#endif
```